ci-operator/step-registry/aws/c2s/get-project-info/aws-c2s-get-project-info-ref.yaml

```diff
@@ -0,0 +1,14 @@
+ref:
+  as: aws-c2s-get-project-info
+  from_image:
+    namespace: ocp
+    name: "4.12"
+    tag: upi-installer
```

Comment:
This step just needs `sed`, there's no reason to use a heavy `upi-installer`? `centos8` should be sufficient imo

Reply:
In the step, it needs `jq`, maybe we can use `cli-jq` image here.
```
  from_image:
    name: cli-jq
    namespace: ocp
    tag: latest
```